README.rst

```diff
@@ -88,37 +88,34 @@ Container Images
 ----------------
 
 Bandit is available as a container image, built within the bandit repository
-using GitHub Actions. The image is available on gchr.io:
+using GitHub Actions. The image is available on ghcr.io:
 
 ```bash
-docker pull gcr.io/pycqa/bandit/bandit:py312-arm64
+docker pull ghcr.io/pycqa/bandit/bandit
 ```
 
 The image is built for the following architectures:
 
 * amd64
 * arm64
+* armv7
+* armv8
 
-The image is tagged with the Python version and architecture, for example:
+To pull a specific architecture, use the following format:
 
-* py312-amd64
-* py312-arm64
-
-Python versions supported are:
+```bash
+docker pull ghcr.io/pycqa/bandit/bandit:<tag>-<arch>
+```
 
-* 3.8 (py38-amd64)
-* 3.9 (py39-amd64)
-* 3.10 (py310-amd64)
-* 3.11 (py311-amd64)
-* 3.12 (py312-amd64)
+Where `<tag>` is the release version of Bandit and `<arch>` is the architecture
```

Comment:
Assuming this is accurate and we want to keep this, `<tag>` here and `<version>` below should probably be the same placeholder rather than something else. Given `tag`'s meaning in the context of a docker image, I'd vote for updating this area to ues `<version>` instead.

Reply:
No, that's wrong I replaced that. Good catch / my bad.

Reply:
This was stale / wrong. Removed it now.